contracts/README.md

```diff
@@ -344,7 +344,7 @@ parameter `initiator` for the `SSNList` contract.
 |`UpdateVerifierRewardAddr(addr: ByStr20)` | `UpdateVerifierRewardAddr(addr: ByStr20, initiator : ByStr20)`|
 |`UpdateStakingParameters(min_stake: Uint128, min_deleg_stake: Uint128, max_comm_change_rate: Uint128)` | `UpdateStakingParameters(min_stake: Uint128, min_deleg_stake: Uint128, max_comm_change_rate: Uint128, initiator : ByStr20) `|
 |`ChangeBNumReq(input_bnum_req: Uint128)` | ` ChangeBNumReq(input_bnum_req: Uint128, initiator : ByStr20)`|
-|`UpdateContractAddr(proxy_address: ByStr20, gzil_address: ByStr20)` | `UpdateContractAddr(proxy_addr: ByStr20, gzil_addr: ByStr20, initiator : ByStr20)`|
+|`UpdateGzilAddr(gzil_address: ByStr20)` | `UpdateGzilAddr(gzil_addr: ByStr20, initiator : ByStr20)`|
```

Comment:
Can we make the parameter name `gzil_addr` consistent across Proxy and SSNList?

Reply:
Yes, updated!